Packs/CommonTypes/pack_metadata.json

```diff
@@ -2,9 +2,9 @@
     "name": "Common Types",
     "description": "This Content Pack will get you up and running in no-time and provide you with the most commonly used incident & indicator fields and types.",
     "support": "xsoar",
-    "currentVersion": "3.2.26",
+    "currentVersion": "3.2.27",
     "author": "Cortex XSOAR",
-    "url": "https://www.paloaltonetworks.com/cortex",
+    "url": "https:\/\/www.paloaltonetworks.com\/cortex",
```

Comment:
@jochman I'm using your version of the SDK with the new JSON loader. Could this be related?

Reply:
that's what happened when you ran format?

Reply:
Also, what is the impact when you're uploading it to XSOAR? If there is any.

Reply:
<img width="802" alt="image" src="https://user-images.githubusercontent.com/11165655/161035629-3c557ee5-67a1-439f-9305-e78ce356fb79.png">
As you can see, the python's built-in is loading it as intended to be.

The only side-effect I can see is if XSOAR isn't parsing the json (go parser) as intended.

Reply:
I assume that the version bump did that and not the format, don't know what the effect would be on XSOAR

Reply:
@dorschw have u seen this before? happened after bumping version

Reply:
It should be okay with the escape character.

Reply:
solved in https://github.com/demisto/demisto-sdk/pull/2005 FYI @idovandijk